final/master.py

```diff
@@ -0,0 +1,408 @@
+import pygame, sys, time, random, math
+from pygame.locals import *
+
+pygame.init()
+pygame.mixer.init()
+
+# Declare all variables etc
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLUE = (0, 0, 255)
+
+WIDTH = 900
+HEIGHT = 600
+
+window = pygame.display.set_mode((WIDTH, HEIGHT))
+
+# Insert picture name to load below
+picture = pygame.image.load('cool_cat.jpg')
+picture = pygame.transform.scale(picture, (WIDTH, HEIGHT))
+
+window.blit(picture, (0, 0))
+
+a = 1
+b = 452
+switch_comparison_direction = 1
+run_once = True
+
+
+# Define functions
+
+
+# Stops the error when a colour value is greater than 255 or less than 0
```

Comment:
Please use [doc-strings](https://www.python.org/dev/peps/pep-0008/#documentation-strings) as according to the PEP-8 and PEP-257 conventions.